data/kitchen-sink/manuscript.xml

```diff
@@ -250,19 +250,166 @@
+        </element-citation>
+      </ref>
+      <!-- Book reference: publisher location - US: city followed by state code -->
+      <ref id="book3">
         <element-citation publication-type="book">
           <person-group person-group-type="author">
             <name>
-              <surname>Council</surname>
-              <given-names>National Research</given-names>
+              <surname>Coyne</surname>
+              <given-names>JA</given-names>
+            </name>
+            <name>
+              <surname>Orr</surname>
+              <given-names>HA</given-names>
+            </name>
+          </person-group>
+          <person-group person-group-type="editor">
+            <name>
+              <surname>Otte</surname>
+              <given-names>D</given-names>
+            </name>
+            <name>
+              <surname>Endler</surname>
+              <given-names>JA</given-names>
+            </name>
+          </person-group>
+          <year iso-8601-date="1989">1989</year>
+          <chapter-title>Two rules of speciation</chapter-title>
+          <source>Speciation and its consequences</source>
+          <fpage>1</fpage>
+          <lpage>2</lpage>
+          <publisher-loc>Sunderland, MA</publisher-loc>
+          <publisher-name>Sinauer Associates</publisher-name>
+        </element-citation>
+      </ref>
+      <!-- Book reference: simple book reference, with ISBN -->
+      <ref id="book4">
+        <element-citation publication-type="book">
+          <person-group person-group-type="author">
+            <name>
+              <surname>Horne</surname>
+              <given-names>P</given-names>
+            </name>
+            <name>
+              <surname>Page</surname>
+              <given-names>J</given-names>
+            </name>
+          </person-group>
+          <year iso-8601-date="2008">2008</year>
+          <source>Integrated pest management for crops and pastures</source>
+          <publisher-name>Landlinks Press</publisher-name>
+          <publisher-loc>Victoria</publisher-loc>
+          <pub-id pub-id-type="isbn">9780643092570</pub-id>
+        </element-citation>
+      </ref>
+      <!-- Book reference: Has volume, no publisher location -->
+      <ref id="book5">
+        <element-citation publication-type="book">
+          <person-group person-group-type="author">
+            <name>
+              <surname>Koch</surname>
+              <given-names>S</given-names>
+            </name>
+          </person-group>
+          <year iso-8601-date="1959">1959</year>
+          <volume>3: Formulations of the Person and the Social Context</volume>
+          <source>Psychology A Study Of A Science</source>
+          <publisher-name>Mcgraw-Hill Book Company</publisher-name>
+        </element-citation>
+      </ref>
+      <!-- Book Chapter reference: chapter in book with editors -->
+      <ref id="chapter1">
+        <element-citation publication-type="chapter">
```

Comment:
@michael Any reason you changed the `publication-type` here compared to what is given as a real example `bib48` in https://github.com/substance/texture/issues/481? Currently both should work, because I added the support for the other one in https://github.com/substance/texture/pull/484/commits/49fbf882c8502eadad6ac0f2e89dc2e0417ec1dc.

Reply:
I thought it may be more explicit to introduce a separate type in JATS as well (less conversion magic). I'd personally be fine with going close to CSL/JSON also on the JATS level, as long as the requirements of the publishers are met.

One question: Does CSL/JSON allow basic formatting (e.g. italics in titles)

Reply:
Ah, I see. Going close to CSL-JSON does sound fine for me, but I guess it needs some discussions with the others.

> One question: Does CSL/JSON allow basic formatting (e.g. italics in titles)

There are some HTML tags which Zotero imports e.g. in title field and can be used for the citations. A recent commit https://github.com/zotero/translators/blob/d1312c62730a23d936a6862f5e2a289420cdd371/Crossref-REST.js#L47-L55 might show the capabilities and you can see an example [here](https://github.com/zotero/translators/blob/d1312c62730a23d936a6862f5e2a289420cdd371/Nature%20Publishing%20Group.js#L1714 ):
```
"title": "Structure of the SAM-II riboswitch bound to <i>S</i>-adenosylmethionine",
```

However, I don't know whether this is part of the specification or just of the (standard) implementation, but I guess it doesn't change much.

Reply:
Absolutely. I'm updating the Dar spec accordingly, and then we can have everyone review the fields. Our strategy for Texture is to only add what is commonly agreed on, and leave out edge cases until they are requested, fully understood and confirmed by everyone. I've moved your other suggestions to separate issues.

As for formatting, I'd allow it for certain fields, but go with the semantical HTML tags. E.g. `em`, `strong` instead of `i` and `b`. We could consider `i` and `b` on import but normalise on export. How does that sound to you?

Will move this into a separate issue too.

Reply:
As far as I can say that sounds fine. Note that `<sup>` and `<sub>` might be relevant e.g. for chemical formulas. Maybe also MathML for mathematical formulas...